src/main/java/com/bandwidth/sdk/BandwidthClient.java

```diff
@@ -65,6 +78,11 @@ public synchronized static BandwidthClient getInstance() {
             String apiEndpoint = env.get(BandwidthConstants.BANDWIDTH_API_ENDPOINT);
             String apiVersion = env.get(BandwidthConstants.BANDWIDTH_API_VERSION);
 
+            // http connection settings
+            String maxTotal = env.get(BandwidthConstants.BANDWIDTH_HTTP_MAX_TOTAL_CONNECTIONS);
```

Comment:
Convert the string values to Integer in this step and throw exception specifying what is wrong